source/material_model/steinberger.cc

```diff
@@ -170,12 +170,17 @@ namespace aspect
     Steinberger<dim>::
     viscosity (const double temperature,
                const double /*pressure*/,
-               const std::vector<double> &,
+               const std::vector<double> &volume_fractions,
                const SymmetricTensor<2,dim> &,
                const Point<dim> &position) const
     {
       const double depth = this->get_geometry_model().depth(position);
       const double adiabatic_temperature = this->get_adiabatic_conditions().temperature(position);
+      // Calculate volume fractions from mass fractions
```

Comment:
Can the following four lines be removed?

Reply:
I am not sure why you wanted to remove these. Can you please elaborate a bit?